app/src/common/shared/com/igalia/wolvic/ui/widgets/KeyboardWidget.java

```diff
@@ -1373,9 +1374,12 @@ public void restartInput(@NonNull WSession session, int reason) {
     }
 
     @Override
-    public void showSoftInput(@NonNull WSession session) {
+    public void showSoftInput(@NonNull WSession session, View requestView) {
+        mExternalFocusedView = requestView;
         if (mFocusedView != mAttachedWindow || getVisibility() != View.VISIBLE || mInputRestarted) {
-            post(() -> updateFocusedView(mAttachedWindow));
```

Comment:
Let's not change this as it's not needed and this is a more compact representation.

Reply:
Done